src/modules/3display.js

```diff
@@ -50,6 +59,20 @@ export default function displayTasks() {
       localStorage.setItem('taskArr', JSON.stringify(parsedArr));
       taskList.removeChild(taskItem);
     });
+
+    /* Checkbox status   */
+    const checkbox = taskItem.querySelector('.checkbox');
+    checkbox.addEventListener('change', () => {
+      task.completed = checkbox.checked;
+      localStorage.setItem('taskArr', JSON.stringify(parsedArr));
+
+      if (task.completed) {
+        updateText.style.textDecoration = 'line-through';
+      } else {
+        updateText.style.textDecoration = 'none';
+      }
+    });
```

Comment:
- [ ] Great job far 👍🏻 clap  According to the project requirements, The user will be able to mark task completion by selecting the corresponding checkbox. The updated tasks list will be stored in local storage. However, The updated array of items is not stored in local storage correctly as shown in the video screen below 👇🏻. Make sure when you reload the page, the user gets the correct list values

Reply:
@BenMukebo can you please send the screen?

Reply:
@BenMukebo I don't understand what you are asking for.
When I click the checkbox, the complete changes from false to true, and then, when I click clear all completed
![image](https://user-images.githubusercontent.com/109246702/193175583-cafda20b-f2d0-4370-933a-9b098d72a29a.png)

Reply:
@BenMukebo the logic I am applying is that the task disappears from the local storage only when I click "clear completed"

Reply:
I am asking for another review because I don´t understand what is he asking for
I have tried to contact him by Github and Slack and it hasn't been possible
I think my project is working correctly

Reply:
Hello @AlejandraKeber !!

Reply:
Let's have a zoom call so that I can explain you clearly about the issue!!